ImageItem/ImageItem.js

```diff
@@ -35,6 +35,14 @@ const ImageItemBase = kind({
 	name: 'ImageItem',
 
 	propTypes: /** @lends agate/ImageItem.ImageItemBase.prototype */ {
+		/**
+		 * Display caption over image.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		captionOverImage: PropTypes.bool,
```

Comment:
As @seunghoh mentioned in https://github.com/enactjs/agate/pull/279#discussion_r447323032, how about providing a `type` prop including `fullimage` or something else instead of providing a boolean type prop?

Reply:
I changed `captionOverImage: PropTypes.bool` with `captionPosition: PropTypes.oneOf(['captionBelow', 'captionOverlay'])` with default: `captionBelow`.
What is your opinion on this approach?

Reply:
Much better~
Only one thing, I'd like to comment..
How about replacing caption with children?
App developers couldn't know the relationship between caption and children.